spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/pom.xml

```diff
@@ -27,6 +27,7 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-lattice-core</artifactId>
 			<version>${spring-cloud-lattice.version}</version>
+			<scope>runtime</scope>
```

Comment:
Because compile > runtime.
But I agree that if our code does not depend on that jat at compile time, it would be better to put this in runtime scope (the same way binders should be runtime scoped as well, btw)